### PR TITLE
fix(InteractionEventForm and subforms):

### PR DESCRIPTION
**Pull request by daniel7764**
-InteractionEventForm and all of the subforms are now refactored to be "prettier"

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/ContactForm/ContactForm.tsx [hunk 1]
```diff
@@ -81,73 +81,75 @@ const ContactForm: React.FC<Props> = (props: Props): JSX.Element => {
...
+                                                  InteractionEventContactFields.PHONE_NUMBER
+                                              )
+                                              }
+                                              onChange={event => onChange(
+                                                  {
+                                                      ...phoneNumber,
+                                                      number: event.target.value,
+                                                  },
+                                                  InteractionEventContactFields.PHONE_NUMBER
+                                              )}
```
Comment (MayaSasson) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/ContactForm/ContactForm.tsx:102]: Why did you add tabs?

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/PlacesAdditionalForms/TransportationAdditionalForms/FlightEventForm.tsx [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 import { useSelector } from 'react-redux';
-import { Autocomplete } from '@material-ui/lab';
 import { useForm } from "react-hook-form";
+import { Autocomplete } from '@material-ui/lab';
+import { Grid, TextField } from '@material-ui/core';
 
 import Country from 'models/Country';
 import useFormStyles from 'styles/formStyles';
 import StoreStateType from 'redux/storeStateType';
+import useStyles from './TransportationFormsStyles';
```
Comment (MayaSasson) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/PlacesAdditionalForms/TransportationAdditionalForms/FlightEventForm.tsx:10]: imports order

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/PlacesAdditionalForms/TransportationAdditionalForms/TrainEventForm.tsx [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
 import React, { useContext } from 'react';
+import { useForm } from 'react-hook-form';
 import { Autocomplete } from '@material-ui/lab';
-import { useForm } from "react-hook-form";
+import { Grid, TextField } from '@material-ui/core';
 
 import City from 'models/City';
 import useFormStyles from 'styles/formStyles';
 import StoreStateType from 'redux/storeStateType';
+import useStyles from './TransportationFormsStyles';
```
Comment (MayaSasson) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/PlacesAdditionalForms/TransportationAdditionalForms/TrainEventForm.tsx:10]: imports order

File: client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetailsStyles.ts [hunk 4]
```diff
@@ -60,12 +65,15 @@ export const useStyles = makeStyles({
...
-        width: '26vw',
+    hospitalInput: {
+      marginLeft: '6vw',
     },
+    // hospitalizedDates: {
+    //     '@media screen and (min-width: 1870px)': {
+    //         width: '14vw',
+    //     },
+    //     width: '26vw',
+    // },
```
Comment (MayaSasson) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetailsStyles.ts:76]: remove

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/BusinessContactForm/BusinessContactForm.tsx [hunk 5]
```diff
@@ -1,8 +1,9 @@
 import React, { useContext } from 'react';
-import { Grid, TextField, Typography } from '@material-ui/core';
-import { useForm } from "react-hook-form";
+import { Grid, Typography } from '@material-ui/core';
+import { useForm } from 'react-hook-form';
 
 import useFormStyles from 'styles/formStyles';
+import useStyles from './BusinessContactFormStyles';
```
Comment (MayaSasson) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/BusinessContactForm/BusinessContactForm.tsx:6]: imports order

**Review by MayaSasson — DISMISSED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/AddressForm/AddressForm.tsx [hunk 6]
```diff
@@ -25,18 +25,13 @@ const AddressForm : React.FC = () : JSX.Element => {
...
-            <Grid container justify='flex-start' className={[formClasses.formRow, additionalClasses.addressRow].join(' ')}>
-                <Grid item xs={6}>
-                    <FormInput fieldName='כתובת'>
-                        <LocationInput selectedAddress={locationAddress}
-                                        setSelectedAddress={onGoogleApiLocationTextFieldChange}/>
-                    </FormInput>
-                </Grid>
-                <Grid item xs={6}/>
+        <Grid container justify='flex-start' className={[formClasses.formRow, additionalClasses.addressRow].join(' ')}>
+            <Grid item xs={2}>
```
Comment (sandymash) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/AddressForm/AddressForm.tsx:29]: maybe a bit bigger? 2 is really small

**Review by eladyashar — APPROVED** [review]

**Review by eladyashar — APPROVED** [review]

**Review by sandymash — APPROVED** [review]
